helm/prometheus-rules/templates/alerting-rules/up.all.rules.yml

```diff
@@ -48,34 +48,16 @@ spec:
       annotations:
         description: '{{`KubeStateMetrics ({{ $labels.instance }}) is down.`}}'
         opsrecipe: kube-state-metrics-down/
-      expr: label_replace(up{app="kube-state-metrics"}, "ip", "$1.$2.$3.$4", "node", "ip-(\\d+)-(\\d+)-(\\d+)-(\\d+).*") == 0
+      expr: label_replace(up{app="kube-state-metrics"}, "ip", "$1.$2.$3.$4", "node", "ip-(\\d+)-(\\d+)-(\\d+)-(\\d+).*") == 0  or absent(up{app="kube-state-metrics"} == 1)
```

Comment:
What's this magic with the node ip ? Do we need to propagate this ?

Reply:
I'm not sure.. .

Reply:
all up alerts are the same

Reply:
Do we merge?

Reply:
I think this label might be used to match against other metrics, so I believe we should keep it.

Reply:
I did not check, but I'd bet on matching inhibitions.

Reply:
@QuentinBisson  has more knowledge about this

Reply:
I confirm it is inhibitions:

```
- source_matchers:
  - kubelet_down="true"
  target_matchers:
  - cancel_if_kubelet_down="true"
  equal:
  - cluster_id
  - ip
```
and:
```
- source_matchers:
  - kubelet_not_ready="true"
  target_matchers:
  - cancel_if_kubelet_not_ready="true"
  equal:
  - cluster_id
  - ip
```